convex_hulls.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
```

Comment:
Please move this file into ulmo/analysis/

Reply:
same goes for several of the other files.

Also, you might put *all* of them into one module.
Call it `curvature.py`